### PR TITLE
Stop building Ubuntu16.10 builds from master ( 2.1.x )

### PR DESCRIPTION
**Pull request by rakeshsinghranchi**

### CODE REVIEW

**Comment by rakeshsinghranchi** [comment]
cc @leecow @Petermarcu

File: buildpipeline/Core-Setup-Linux-BT.json [hunk 1]
```diff
@@ -157,7 +157,7 @@
...
+        "arguments": "run --privileged --rm $(DockerCommonRunArgs) $(PB_GitDirectory)/Tools/msbuild.sh $(PB_GitDirectory)/publish/publish.proj /p:AzureAccountName=$(PB_AzureAccountName) /p:ContainerName=$(PB_ContainerName) /p:AzureAccessToken=$(PB_AzureAccessToken) /p:ChecksumAzureAccountName=$(PB_ChecksumAzureAccountName) /p:ChecksumContainerName=$(PB_ChecksumContainerName) /p:ChecksumAzureAccessToken=$(PB_ChecksumAzureAccessToken) /p:TargetArchitecture=$(PB_TargetArchitecture) /p:ConfigurationGroup=$(BuildConfiguration) /p:PortableBuild=$(PB_PortableBuild) /p:OSGroup=Linux /p:DebRepoUser=$(PB_DebRepoUser) /p:DebRepoServer=$(PB_DebRepoServer) /p:DebRepoPass=$(DEB_REPO_PASSWORD) /p:DebianId_ubuntu1404-x64=$(PB_DebianId_ubuntu1404-x64) /p:DebianId_debian8-x64=$(PB_DebianId_debian8-x64) /p:DebianId_ubuntu1604-x64=$(PB_DebianId_ubuntu1604-x64)",
```
Comment (eerhardt) [inline buildpipeline/Core-Setup-Linux-BT.json]: This isn't about Ubuntu16.10 specifically, but why don't we have Ubuntu17 in this list?  

Looking at what it is doing - this step is really publishing for RHEL7, right?  Do we even need this list of Debian Ids on this step?

**Comment by rakeshsinghranchi** [comment]
Makes sense, removed the unwanted Debian keys . Also, opened an issue #3369 to clean up other variables that are not used anymore.

**Comment by eerhardt** [comment]
Thanks!

**Pull request merged**